src/Rules/AnyOfRule.php

```diff
@@ -12,13 +12,14 @@ final class AnyOfRule extends CompoundRule
 {
 
 	/**
-	 * @param mixed $value
+	 * @param mixed            $value
 	 * @param CompoundRuleArgs $args
 	 * @return mixed
 	 * @throws ValueDoesNotMatch
 	 */
 	public function processValue($value, Args $args, FieldContext $context)
 	{
+		$inputValue = $value;
```

Comment:
It shouldn't be needed here, it just needs to be tested

Reply:
TODO: test case

Reply:
Test case added.